### DISCUSSION TITLE
dynamic Routes gets this value 4f64eaa44708eacdfb67703150ce5f05.jpg

### DISCUSSION THREAD

**Discussion by Mohammad-Shahid-07** [vercel/next.js#60805] [App Router]
so whenever i create a dynamic route using app router like this [id]/page.ts 
sometimes the params get this value "4f64eaa44708eacdfb67703150ce5f05.jpg" this has happened on different projects . does this happen with everyone??

**Comment by icyJoseph** [comment]
Do you see this in incognito mode? Or perhaps you used a Service worker at some point? Do you see it in production? 

I think here, quite simply, the `[id]/page.tsx` route is catching some spurious request made into your server.
